spec/features/categories_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
```

Comment:
- Please, This is what I am getting: "rspec spec"
- Make sure that all the test pass.

![Capture](https://user-images.githubusercontent.com/63972670/196382859-70b8cd99-860e-4a7b-9af9-b6ebb6b4e2f8.JPG)